models/yang/sonic/sonic-show-techsupport.yang

```diff
@@ -0,0 +1,40 @@
+module sonic-show-techsupport {
+	namespace "http://github.com/Azure/sonic-show-techsupport";
+	prefix sshwtchspt;
+    yang-version 1.1;
+
+	import ietf-yang-types {
+		prefix yang;
+	}
+
+	import ietf-inet-types {
+                prefix inet;
+    }
+
+	organization
+		"BRCM";
```

Comment:
Update to "SONiC"

Reply:
I have updated this and other similar instances. Thank you for providing your advice on this.